pkg/ibmvpc/examples/out/explain_out/LBToResIPNode_all_vpcs_explain_debug.txt

```diff
@@ -4,7 +4,7 @@ Explaining connectivity from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca t
 No connections are allowed from kube-clusterid:1-8fdd1d0a2ce34deba99d0f885451b1ca[LB private IP][192.168.36.6] to iks-clusterid:1[192.168.32.5];
 connection is blocked by Load Balancer
```

Comment:
this phrasing is also an issue for LB , connection is not "blocked" by load balancer , and the term "no connections are allowed" is also an issue for LB context

Reply:
I would replace "no connections are allowed from .." with just "no connections from ..." 

and remove the sentence "connection is blocked by Load Balancer"

Reply:
for the other LB example  - would replace "Allowed connections from .. " to "connections from ... "